model/src/main/kotlin/Vulnerability.kt

```diff
@@ -43,4 +43,37 @@ data class Vulnerability(
      */
     @JsonInclude(JsonInclude.Include.NON_NULL)
     val url: URL? = null
-)
+) {
+    /**
+     * See https://www.first.org/cvss/specification-document#Qualitative-Severity-Rating-Scale.
+     */
+    enum class Rating(private val start: Float, private val endExclusive: Float) {
+        NONE(0.0f, 0.1f),
+        LOW(0.1f, 4.0f),
+        MEDIUM(4.0f, 7.0f),
+        HIGH(7.0f, 9.0f),
+        CRITICAL(9.0f, 10.0f);
```

Comment:
The critical range includes 10 but the implementation doesn't. Would it make sense to fix that?

Reply:
Good point, fixed.